src/libpmem2/map_windows.c

```diff
@@ -228,7 +228,12 @@ pmem2_map(const struct pmem2_config *cfg, struct pmem2_map **map_ptr)
 		goto err_unmap_base;
 
 	map->addr = base;
-	map->length = length;
+	/*
+	 * XXX probably in some cases the reserved length > the content length.
+	 * Maybe it is worth to do the research.
```

Comment:
are you going to update this in this PR?

Reply:
Yea, don't add TODOs without stating in the comment what it's waiting for.